python/rules_engine.py

```diff
@@ -2644,6 +2643,136 @@ def v708(self):
 		fail_df = self.lar_df[~(self.lar_df.business_purpose.isin(("1111", "1", "2")))]
 		self.results_wrapper(edit_name=edit_name, field_name=field, fail_df=fail_df)
 
+	def v709(self):
+		"""An invalid Property Address was reported. Please
+		review the information below and update your file
+		accordingly.
+		1) If Street Address, City, and Zip Code is reported
+		Exempt, then all three must be reported Exempt. """
+
+		field = "Property Address"
+		edit_name = "v709"
+		fail_df = self.lar_df[((self.lar_df.street_address=="Exempt")&(self.lar_df.city=="Exempt")&(self.lar_df.zip_code == "Exempt"))&
+			((self.lar_df.street_address!="Exempt") & (self.lar_df.city !="Exempt") & (self.lar_df.zip_code != "Exempt"))]
+		self.results_wrapper(edit_name=edit_name, field_name=field, fail_df=fail_df)
+
+	def v710_1(self):
+		"""If the Credit Score exemption election is taken,
+			1) Credit Score of Applicant or Borrower, Credit
+			Score of Co-Applicant or Co-Borrower, Applicant or
+			Borrower, Name and Version of Credit Scoring
+			Model, and Co-Applicant or Co-Borrower, Name and
+			Version of Credit Scoring Model must be reported
+			1111.""" 
+		field = "Credit Score"
+		edit_name = "v710_1"
+		fail_df = self.lar_df[(self.lar_df.app_credit_score=="1111")&((self.lar_df.co_app_credit_score!="1111")& 
+				(self.lar_df.app_score_name!="1111")&(self.lar_df.co_app_score_name!="1111"))]
+		self.results_wrapper(edit_name=edit_name, field_name=field, fail_df=fail_df)
+	
+	def v710_2(self):
+		"""If the Credit Score exemption election is taken,
+			2) Applicant or Borrower, Name and Version of
+				Credit Scoring Model: Conditional Free Form Text
+				Field for Code 8 and Co-Applicant or Co-Borrower,
+				Name and Version of Credit Scoring Model:
+				Conditional Free Form Text Field for Code 8 must be
+				left blank."""
+		field = "Credit Score"
+		edit_name = "v710_2"
+		fail_df = self.lar_df[(self.lar_df.app_credit_score=="1111")&((self.lar_df.app_score_name!="")& 
```

Comment:
The second part of the boolean logic may need checking. Is it producing the correct results for edit fails? I think the ors might need to be ands.